src/bundle/Resources/views/content/tab/urls.html.twig

```diff
@@ -57,19 +57,34 @@
     }) }}
 {% endif %}
 
-{% include '@ezdesign/content/tab/url/modal_add_custom_url.html.twig' with {'form': form_custom_url_add, 'parent_name': parent_name} only %}
+{% if can_edit_custom_url %}
+    {% include '@ezdesign/content/tab/url/modal_add_custom_url.html.twig' with {
+        'form': form_custom_url_add,
+        'parent_name': parent_name
+    } only %}
+{% endif %}
 
 {% macro table_header_tools(form_custom_url_remove, can_edit_custom_url) %}
     {% if can_edit_custom_url %}
-        <button class="btn btn-primary ez-btn--prevented" data-toggle="modal" data-target="#ez-modal--custom-url-alias" title="{{ 'tab.urls.action.add'|trans|desc('Add Custom URL') }}">
+        <button
+            class="btn btn-primary ez-btn--prevented"
```

Comment:
`ez-btn--prevented` should come along with `ez-btn` class name as well.

Reply:
I have only reformated the code here for better readability and I think it is not the scope of this bug

Reply:
I agree, let's not mess with it now as it might break things.